test/MockSmartSyncPlugin.js

```diff
@@ -155,18 +155,43 @@ var MockSmartSyncPlugin = (function(window) {
                     fieldlist: options.fieldlist,
                     cache: cache,
                     cacheMode: Force.CACHE_MODE.SERVER_FIRST,
-                    mergeMode: options.mergeMode,
+                    mergeMode: Force.MERGE_MODE.OVERWRITE,
```

Comment:
smartsync.js merge modes involve an original copy and fields diffing, so we can't use it here